test/fail/ok/object-ext-pseudo.tc.ok

```diff
@@ -0,0 +1,2 @@
+object-ext-pseudo.mo:1.16-1.27: type error [M0093], expected object type, but expression produces type
```

Comment:
I am expecting 2 error messages!

maybe I am calling the wrong error reporting function!?

Reply:
I'm inclined to experiment with this in a separate PR.

Reply:
It could be local_error vs error (IIRC)

Reply:
It could be because the error isn't an an analysis context, but in inference, so it can't recover and proceed. If you add a type annotation, do you get more errors?